src/components/DayPickerNavigation.jsx

```diff
@@ -122,21 +122,21 @@ function DayPickerNavigation({
           {...css(
             styles.DayPickerNavigation_button,
             isDefaultNavPrev && styles.DayPickerNavigation_button__default,
-            ...(isHorizontal && [
+            ...(isHorizontal ? [
               styles.DayPickerNavigation_button__horizontal,
-              ...isDefaultNavPrev && [
+              ...(isDefaultNavPrev ? [
```

Comment:
well this is annoying. can I no longer spread on falsey things?

Reply:
apparently you never could for non-objects, but Babel mistakenly allowed it.

https://github.com/babel/babel/pull/7491

Reply:
weird okay!